### ISSUE TITLE
Define the object of each WCAG test

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#124]
Each WCAG test is focused on a certain object that is tested. This is most of the cases an HTML-document, but this is sometimes a part of the web page:
- a table
- a form
- a link
- an image
  etc. 

This is described in the applicability of a test. Sadly this is not everywhere well defined. So we need to define our own scope, preferrably scaled down to a html element so that we can detect this element with the selector.
We need this to check if a test is passed, failed or is not applicable, and to do calculations. Best would be an analysis for each test to define the scope in a sheet or wiki page. Afterwards we can implement this in the tests.

see also #101
